addon/data/shooter-interactive-worker.js

```diff
@@ -241,6 +251,10 @@ function restore(state, pos, scrollIntoView) {
       window.scrollTo(0, newScroll);
     }
   }
+
+  if (state === "text") {
+    addHandlers();
```

Comment:
You're right. I thought it was required, but I just tested without it and it's not.